modules/ztp-image-based-upgrade-shared-container-partition.adoc

```diff
@@ -14,7 +14,7 @@ You must complete this procedure at installation time.
 
 .Prerequisites
 
-* Install Butane.
+* Install Butane. For more information, see "Installing Butane".
```

Comment:
This seems like a text (not link). Will it work?

Reply:
The working link is in the "Additional resources" at the end of [this page](https://78655--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.html) after the section. Unfortunately, this is a technical limitation